src/modules/users/dto/newPassword.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty, IsString, Length, MinLength } from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class NewPasswordDto {
+  @IsNotEmpty()
+  @Type(() => String)
```

Comment:
Why do we need this here?

Reply:
Somebody will send 123456

Reply:
if someone sends 123456, then request body is not valid and the sender will receive an error

Reply:
But we can just change password (